contents/teams/sales-cs/objectives.mdx

```diff
@@ -4,16 +4,17 @@
     - Add 3 extra large customers ($100k/yr) - _Simon_
     - Add 4 large customers ($60k/yr) - _Cameron_
     - Add 20 medium customers ($20k/yr) - _Cameron_
-    - Implement new automations that increase conversion from demo to paying, and potentially engage more self serve people - _Joe_
+    - Implement new automations that increase conversion from demo to paying - _Joe_
   - Hit monthly logo retention targets for customers who have paid 2+ invoices:
     - Extra large customers ($100k/yr) - 100% - _Simon_
-    - Large customers ($60k/yr) - 95% - _Mine_
-    - Medium customers ($20k/yr) - 90% - _Mine_
-    - Build a retention playbook so we have a repeatable process - _Mine_
-  - Hire a CSM and an inbound SDR - _Charles_
+    - Large customers ($60k/yr) - 95% - _Cameron_
```

Comment:
I'm conflicted about what to do here - the primary goal of the Account Exec role is Land/Expand.  Are we taking away from that by having a focus on Retention for them here?

Reply:
Maybe have explicit revenue goals per segment?  E.g. the 20k segment starts the quarter at 1.2m ARR and finishes at 1.5m?

Reply:
Have set these based on [this dashboard](http://metabase-prod-us/dashboard/68-nrr-high-pay) (thanks @minekansu!) - look ok to you?

I went for a slight stretch for Q2 at least as I think our focus should lean slightly more towards new revenue, but in future these could be more ambitious.

Reply:
Having discussed further with Simon, we can simplify this a bit so I've gone for a new revenue figure instead, as this effectively combines totally new and expansion revenue into one number, which should help us focus. This is based on our overall Q2 goal, divided proportionally based on each person's book of business (and then we can easily add in another line for the new AE once they join).